src/jenkins_job_insight/llm_pricing.py

```
@@ -0,0 +1,272 @@
+            self._refresh_task.cancel()
+            self._refresh_task = None
+
+    async def _refresh_loop(self) -> None:
+        """Periodically refresh pricing data. Never raises."""
+        while True:
+            try:
+                await asyncio.sleep(_REFRESH_INTERVAL_SECONDS)
+                await self.refresh()
+            except asyncio.CancelledError:
+                break
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)